hasjob/templates/detail.html

```diff
@@ -456,6 +449,23 @@
       return false;
     });
   });
+  $(function () {
+    var triggered_related = false;
+    $(window).scroll(function() {
+      if (!triggered_related &&
+          $(window).scrollTop() >= ($('#stickie-area').position().top - $(window).height()) &&
+          ($(window).height() + $(window).scrollTop()) < $(document).height()
+      ) {
```

Comment:
As an alternative you could try using the `appear` plugin, that we're using to [paginate posts through ajax](https://github.com/hasgeek/hasjob/blob/a7e8a4c769963a0877359297c5661b240c34b310/hasjob/static/js/app.js#L151).